app-arch/tar/tar-1.29.recipe

```diff
@@ -26,17 +26,33 @@ PROVIDES="
 REQUIRES="
 	haiku
 	"
+
 BUILD_REQUIRES="
 	haiku_devel
 	"
 BUILD_PREREQUIRES="
 	cmd:automake
 	cmd:autoreconf
+	cmd:find
 	cmd:gcc
 	cmd:ld
 	cmd:make
 	cmd:sed
 	"
+if [ "$recipeAction" = TEST ]; then
+	BUILD_PREREQUIRES="$BUILD_PREREQUIRES
```

Comment:
It makes me think that it would be cool to have TEST_PREREQUIRES in Haikuporter...